### PR TITLE
Adds elastic6 connector support

### PR DESCRIPTION
**Pull request by caiooliveiraeti**
Adds elastic6 connector support #369

### CODE REVIEW

File: kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  logger.info("Initialising Elastic Json writer")
+
+  //initialize error tracker
+  initialize(settings.taskRetries, settings.errorPolicy)
+
+  //create the index automatically if it was set to do so
+  settings.kcqls.filter(_.isAutoCreate).foreach(client.index)
+
+  settings.kcqls.filter(_.getWriteMode == WriteModeEnum.UPSERT).foreach { kcql =>
+    if (kcql.getPrimaryKeys.size() != 1) {
```
Comment (stheppi) [inline kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: should we allow multiple PK fields and they form the ES key by combining them witha  a separator? (separator can be `.`  - configurable)?

Comment (stheppi) [reply kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: Maybe to the same for es5? @Antwnis @andrewstevenson

File: kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+  /**
+    * Create a bulk index statement and execute against elastic4s client
+    *
+    * @param records A list of SinkRecords
+    **/
+  def insert(records: Map[String, Vector[SinkRecord]]): Unit = {
+    val fut = records.flatMap {
+      case (topic, sinkRecords) =>
+        val kcqls = topicKcqlMap.getOrElse(topic, throw new IllegalArgumentException(s"$topic hasn't been configured in KCQL"))
```
Comment (stheppi) [inline kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: Can we have in the message the configured topics? (do the same for es5)

File: kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala (hunk 1 above)
Comment (caiooliveiraeti) [reply kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: Hi @stheppi, I will work for this.

Comment (caiooliveiraeti) [reply kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: Can I change es5 too?

Comment (andrewstevenson) [reply kafka-connect-elastic6/src/main/scala/com/datamountaineer/streamreactor/connect/elastic6/ElasticJsonWriter.scala]: @caiooliveiraeti Sure do es5 :-)

**Comment by caiooliveiraeti** [comment]
@stheppi If have more fix please send to me. ;-)

**Comment by andrewstevenson** [comment]
@caiooliveiraeti Do you fancy this one? #265

**Pull request merged**